backend/src/main/java/com/pigma/harusari/alarm/command/entity/Alarm.java

```diff
@@ -0,0 +1,42 @@
+package com.pigma.harusari.alarm.command.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "alarm")
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```
Access레벨 protected로 수정해주심 될 것 같습니다